labelme/app.py

```diff
@@ -867,7 +868,7 @@ def __init__(
         if filename is not None and osp.isdir(filename):
             self.importDirImages(filename, load=False)
         else:
-            self.filename = filename
+            self.filename:str|types.NoneType = filename
```

Comment:
can you add `from __future__ import annotations` so that this is compatible with python3.9